controllers/modals/editorDecision/form/EditorDecisionWithEmailForm.inc.php

```diff
@@ -227,7 +254,7 @@ function _sendReviewMailToAuthor($submission, $emailKey, $request) {
 			$reviewIndexes[-1] = $lastIndex + 1;
 
 			// Attach the selected reviewer attachments to the email.
-			$submissionFileDao = DAORegistry::getDAO('SubmissionFileDAO'); /* @var $submissionFileDao SubmissionFileDAO */
```

Comment:
Likewise for these changes -- is there a particular editor/tool with standards you're needing to meet? I think the current formatting is used by VSCode (I don't use it) and PHP Scrutinizer (which is part of our CI stack). Not sure if they like the changed forms.

Reply:
FYI I've been using vscode, and I'm probably using the [same extension](https://marketplace.visualstudio.com/items?itemName=bmewburn.vscode-intelephense-client) as Nate. Perhaps there's a configuration to setup the "phpdoc style"... It isn't recognizing the previous syntax by default.